mitxpro/settings.py

```diff
@@ -624,3 +624,4 @@ def get_all_config_keys():
 
 # Hubspot sync settings
 HUBSPOT_API_KEY = get_string("HUBSPOT_API_KEY", None)
+HUBSPOT_ID_PREFIX = get_string("HUBSPOT_ID_PREFIX", None)
```

Comment:
Everything seems to work, but I think it would be best to provide a default value here for HUBSPOT_ID_PREFIX so we're not sending ids like `None-1` if the setting isn't configured.

```HUBSPOT_ID_PREFIX = get_string("HUBSPOT_ID_PREFIX", 'xpronew')```